.github/workflows/php-code-scanning.yml

```diff
@@ -0,0 +1,44 @@
+name: PHP Code Scanning
+
+on:
+  push:
+    branches:
+      - '*'  # Trigger on commits to any branch
+  pull_request:
+    branches:
+      - 'master'  # Trigger for PRs targeting the 'master' branch
+
+jobs:
+  phpstan:
+    runs-on: ubuntu-latest
```

Comment:
Better pick an explicit version. PHP 7.4 is most certainly not available on every `ubuntu-latest` in the future.